docs/topics/mta-rn-known-issues-7-0-0.adoc

```diff
@@ -35,6 +35,81 @@ The system repeatedly shows a warning message about overriding an inherited asse
 
 This warning, appropriate for the first assessment, incorrectly reappears on subsequent reassessments, suggesting that the application is still inheriting its assessment, even after it has been overridden. link:https://issues.redhat.com/browse/MTA-1825[MTA-1825]
 
+.Custom rule for javax.* package import is not triggered when uploaded as an XML file
+
+In {ProductShortName} 7.0.0, the XML custom rule files are converted to the new YAML format during analysis. However, when using `javax-to-jakarta-package` as a custom rules file, it is not triggered after conversion
+
```

Comment:
In the following example XML rule, note that `when` condition is `javaclass references` ="javax.{*}"`. 

If you amend this to `javaclass references="javax{*}"`, the rule converts correctly.